evaluate/task/task-code-repair_test.go

```diff
@@ -267,3 +269,206 @@ func TestTaskCodeRepairRun(t *testing.T) {
 		}
 	})
 }
+
+func TestValidateCodeRepairRepository(t *testing.T) {
+	validate := func(t *testing.T, tc *tasktesting.TestCaseValidateRepository) {
+		t.Run(tc.Name, func(t *testing.T) {
+			tc.Validate(t, validateCodeRepairRepository)
+		})
+	}
```

Comment:
Please move the `t.Run` into the actual validation function. That way you don't need to define your own here.